config/routes.rb

```diff
@@ -41,6 +41,7 @@
       get 'submission_status/:job_id', to: 'disability_compensation_forms#submission_status', as: 'submission_status'
       get 'user_submissions'
       get 'suggested_conditions'
+      post 'find_rating_info_pid'
```

Comment:
This whole resource is pretty non-RESTy, so makes sense that its just another url, but wondering if something more RESTy would work:

```
resource :disability_compensation_form, only: [] do
  resource :rating, only: :show
end
```

if that makes sense semantically.